packages/devtools_app/lib/src/service/vm_flags.dart

```diff
@@ -16,6 +16,9 @@ const profiler = 'profiler';
 // Defined in SDK: https://github.com/dart-lang/sdk/blob/master/runtime/vm/profiler.cc#L36
 const profilePeriod = 'profile_period';
 
+// Defined in SDK: https://github.com/dart-lang/sdk/blob/main/runtime/vm/microtask_mirror_queues.cc#L18-L23.
```

Comment:
I'm not sure it's worth including the line numbers here, especially if we're using a URL that doesn't point to a specific commit.

Reply:
Done.